__DEFINES/global.dm

```diff
@@ -469,3 +469,13 @@ var/list/variables_not_to_be_copied = list(
 	"on_explode","on_projectile","in_chamber","power_supply","contents",
 	"x","y","z"
 )
+
+// http://ss13.moe/wiki/index.php/Chain_of_Command
+var/list/chain_of_command = list(
+	"Captain" = 1,
+	"Head of Personnel" = 2,
```

Comment:
I would personally put HoPs at the same level of HoSes but I'm slightly biased

Reply:
http://ss13.moe/wiki/index.php/Chain_of_Command

Reply:
Outdated (literal) bay article that doesn't really take into account the concrete reality of the server if you ask me.

Reply:
i don't

Reply:
That's just shifty's headcanon